packages/web3-types/package.json

```
@@ -1,6 +1,6 @@
 {
 	"name": "web3-types",
-	"version": "1.8.0",
+	"version": "1.9.0",
```

```suggestion
	"version": "2.0.0",
```
https://github.com/web3/web3.js/pull/7291#discussion_r1783333683

---

pls see if its not effecting major of web3, else will need adjustment of https://github.com/web3/web3.js/pull/7291/files#diff-f0e2ce27ab978e65f8a25391cd47c34cbc74bf7716a6a870b9fc4c894a39b7d1L518